pkg/sbom/sbom.go

```diff
@@ -58,4 +59,5 @@ type ScanResult struct {
 	Report    Report
```

Comment:
Should we use a pointer to avoir report copy ? 

```suggestion
	Report    *Report
```

Reply:
The report is an interface and the object is already a pointer